packages/components/src/components/step-group/step-group.ts

```diff
@@ -0,0 +1,117 @@
+import { css, html } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { property, query } from 'lit/decorators.js';
+import { watch } from '../../internal/watch.js';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+import type SdStep from '../step/step';
+
+/**
+ * @summary Used as navigation bar that guides users through the steps of a process or task.
+ * @documentation https://solid.union-investment.com/[storybook-link]/step-group
+ * @status stable
+ * @since 3.2.0
+ *
+ *
+ * @slot - Used for grouping steps in the step group. Must be `<sd-step>` elements.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart body - The container that wraps the steps.
+ */
+@customElement('sd-step-group')
+export default class SdStepGroup extends SolidElement {
+  private steps: SdStep[] = [];
+
+  @query('[part=body]') body: HTMLSlotElement;
+
+  /** The step-groups's size. */
+  @property({ reflect: true }) size: 'lg' | 'sm' = 'lg';
+
+  /** Determines the orientation of the step-group. */
+  @property({ reflect: true }) orientation: 'horizontal' | 'vertical' = 'horizontal';
+
+  /** The active step in the step-group. */
+  @property({ type: Number, reflect: true, attribute: 'active-step' }) activeStep = 0;
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.updateComplete.then(() => {
+      this.steps = this.getAllSteps();
+
+      // Remove the tail from the last step
+      this.steps[this.steps.length - 1].noTail = true;
+
+      this.steps.forEach((step, index) => {
+        // Initialize the step attributes
+        step.StepIndex = index + 1;
+        step.size = this.size;
+        step.orientation = this.orientation;
+
+        // Set the active step on click
+        step.addEventListener('click', () => {
+          if (step.state === 'finished') {
+            this.setActiveStep(index);
+          }
+        });
```

Comment:
This should definitely be removed as it is way to invasive. We should never handle the interaction here, as there will usually state management that is in charge of that. It could even happen, that there is data, that is not correct etc. or they shouldn't be allowed to go backt or whatever.

Reply:
@mariohamann We have the requirement that "finished" steps should be clickable. 
from #304 ->
 "Steps that are completed must be clickable. The state from the previous one should also change back to waiting (check the click bahaviour here: https://ant.design/components/steps#steps-demo-clickable)"
 
 The ant.design demo moves to the clicked step. Are you proposing we remove the functionality as a whole?